src/modules/spaces/controllers/spaces.controller.spec.ts

```diff
@@ -94,19 +100,19 @@ describe('SpacesController', () => {
 		it('should return all spaces', async () => {
 			mockSpacesService.findAll.mockResolvedValueOnce(mockSpacesResponse);
 			const spaces = await spacesController.getSpaces({}, new SessionUserEntity(mockUser));
-			expect(spaces.items.length).toEqual(2);
+			expect(spaces.items.length).toEqual(3);
```

Comment:
If we use the length of the mock array, we never have to update this again. And it explains where the "3" value comes from
```suggestion
			expect(spaces.items.length).toEqual(mockSpacesResponse.items.length);
```